nas_spec/components/schemas/issuing/transaction-simulator/authorizations/SimulateAuthorizationResponse.yaml

```
@@ -0,0 +1,16 @@
+type: object
+description: Authorization Response.
+required:
+  - status
+properties:
+  id:
+    type: string
+    description: The unique identifier of the transaction, linking together all related authorization, reversal and clearing messages.
+    example: trx_aaqc4uaaybigcaaqc4uaayfiga
+  status:
+    type: string
+    description: The status of the transaction (i.e. Authorized or Declined).
```

```suggestion
    description: The status of the transaction (for example, Authorized or Declined).
```
Reason for change: screen readers can misread 'e.g.' as 'egg'. 🥚